autometa/taxonomy/ncbi.py

```diff
@@ -399,12 +426,12 @@ def main():
     parser.add_argument('ncbi', help='</path/to/ncbi/database/directory>')
     parser.add_argument(
         '--query-taxid',
-        help='query taxid to test NCBI class functionality',
+        help='query taxid to test NCBI class (i.e. this script\'s) functionality',
```

Comment:
What functionality? What will it actually run (in terms of function etc.) If a user doesn't know this, how will they know of the "test" is giving the expected result?

Reply:
Do we want to be able to run `ncbi.py` as a standalone module, and if so what functionality should be included?


### Ideas if we want to run as a standalone module

* Writing a dataframe of canonical ranks give a query set of taxids
* common ancestry check
* lineage query
* old vs. new taxid query (i.e. is the taxid found in `merged.dmp` or in `nodes.dmp`?)

Reply:
As of now am mentioning the output that is being printed by `logger.INFO` in the argparse block. 
If we decide to run it as a standalone module I will modify it as per the suggestions by @jason-c-kwan and @WiscEvan

Reply:
If this is not run as a standalone module. You can probably just print this information, or remove `argparse` entirely